python/sglang/srt/managers/router/model_runner.py

```diff
@@ -40,6 +40,10 @@ def import_model_classes():
     package = importlib.import_module(package_name)
     for _, name, ispkg in pkgutil.iter_modules(package.__path__, package_name + "."):
         if not ispkg:
+            # gemma references InputMetadata that does not exist, skip it for now.
+            # TODO Wait for sglang/main to fix it.
+            if name in ["sglang.srt.models.gemma"]:
+                continue
```

Comment:
What are the problems? Could you tell me how to fix it?

Reply:
@ZhouXingg